### ISSUE TITLE
Create new streaming endpoint dialog box is broken - too big

### ISSUE THREAD

**Issue by MinshikPark** [Azure/Azure-Media-Services-Explorer#135]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue concerns: (mark with an `x`)
```
- [ ] AMSE v4.x (for AMS v2)
- [x] AMSE v5.x (for AMS v3)
```

### This issue is for a: (mark with an `x`)
```
- [ ] bug report
- [x] UI issue report
- [ ] feature request
- [ ] performance problem
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
>Launch latest AMSE for v3 Version 5.2.0.0
>Connect to media account
>Go to Streaming endpoints tab
>Right mouse click and select "Create streaming endpoint.."
>Streaming endpoint creation dialog pops up with full screen and it's too big.
  Can't select standard/premium settings

### Any log messages given by the failure
>N/A

### Expected/desired behavior
>Dialog should be shown in right size and show all the options for streaming endpoint creation.

### OS and Version?
> Windows 10.

### Versions
>Version 1909 (OS Build 18363.900)

### Mention any other details that might be useful

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

**Comment by xpouyat** [comment]
Thanks for the report. The issue is fixed in the v5.2.0.2 https://github.com/Azure/Azure-Media-Services-Explorer/releases/tag/v5.2.0.2

**Comment by MinshikPark** [comment]
Thanks, Verified at v5.2.0.2
